PurchasesTests/SubscriberAttributes/SubscriberAttributesManagerTests.swift

```diff
@@ -531,7 +531,7 @@ class SubscriberAttributesManagerTests: XCTestCase {
         ]
         mockDeviceCache.stubbedUnsyncedAttributesForAllUsersResult = allAttributes
 
-        let mockError = NSError(domain: Purchases.ErrorDomain, code: 123, userInfo: [:])
+        let mockError = NSError(domain: RCPurchasesErrorCodeDomain, code: 123, userInfo: [:])
```

Comment:
this name is auto-generated, right? there's no way to expose our own naming?

Reply:
That's correct, it's auto-generated. We can rename `ErrorCode` and it will adjust the name, but basically we're stuck with `RCPurchasesErrorCodeDomain`